tst/standard/examples.tst

```diff
@@ -226,6 +226,18 @@ Error, the arguments <n> and <k> must be non-negative integers,
 gap> JohnsonDigraph(IsMutableDigraph, 4, 2);
 <mutable digraph with 6 vertices, 24 edges>
 
+#  SquareGridGraph
+gap> SquareGridGraph(5, 5);
+<immutable symmetric digraph with 25 vertices, 80 edges>
+gap> SquareGridGraph(IsMutable, 5, 5);
+<mutable digraph with 25 vertices, 80 edges>
```

Comment:
A bit more variation is a good idea (and moreover, variation from the manual example):
```suggestion
gap> SquareGridGraph(7, 7);
<immutable bipartite symmetric digraph with bicomponent sizes 25 and 24>
gap> SquareGridGraph(IsMutable, 5, 3);
<mutable digraph with 15 vertices, 44 edges>
```